server/container/guice/cassandra-guice/src/test/java/org/apache/james/CacheSessionTest.java

```diff
@@ -79,7 +78,7 @@ public String checkName() {
         .extension(new DockerElasticSearchExtension())
         .extension(new CassandraExtension())
         .server(configuration -> GuiceJamesServer.forConfiguration(configuration)
-            .combineWith(ALL_BUT_JMX_CASSANDRA_MODULE)
+            .combineWith(ALL_BUT_JMX_CASSANDRA_MODULE, new CassandraCacheSessionModule())
```

Comment:
we don't use it for now, do you mean to merge that or it's just to ensure it would work if used?

Reply:
IMO we can merge as we add the possibility for existing products to use the cache keyspace if they want, while leaving others unaffected.

Reply:
This prevent us from checking the product works without the Cache module, don't you think?